### DISCUSSION TITLE
Welcome to xplora_watch Discussions!

### DISCUSSION THREAD

**Discussion by Ludy87** [Ludy87/xplora_watch#89] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NoLifer88** [comment]
Hi, then I'm trying to install it I'm getting an error Config flow could not be loaded: {"message":"Invalid handler specified"}

What I'm doing wrong?
